src/MetricsUploader/MetricsUploaderWindows.cpp

```diff
@@ -65,9 +58,6 @@ MetricsUploader::~MetricsUploader() {
       ERROR("Error while closing connection: %s", GetErrorMessage(result));
     }
   }
-  if (nullptr != metrics_uploader_client_dll_) {
-    FreeLibrary(metrics_uploader_client_dll_);
-  }
```

Comment:
The code should stay here. FreeLibrary should be commented and a comment should be added explaining why we can't call it.

Reply:
Updated the code: commented FreeLibrary calls and add explanation in the destructor as well